__tests__/index.test.js

```diff
@@ -254,7 +275,127 @@ describe('/api/articles/:article_id/comments', () => {
             })
     });
 
- })
+    test('sends an appropriate status and error message when body field is missing', () => {
+
+        const newComment = {username: 'icellusedkars'}
+
+        return request(app)
+           .post('/api/articles/9/comments')
+            .send(newComment)
+            .expect(400)
+            .then((response) => {
+                expect(response.body.msg).toBe("missing field - body");
+            });
+    });
+
+})
+
+describe('PATCH /api/articles/:article_id', () => {
+    test('PATCHES an article with an updated votes property - status 201', () => {
+
+        const voteUpdate = { inc_votes: 5 }
+
+        const returnedArticle = {
+            article_id: 1,
+            title: 'Living in the shadow of a great man',
+            topic: 'mitch',
+            author: 'butter_bridge',
+            body: 'I find this existence challenging',
+            created_at: '2020-07-09T20:11:00.000Z',
+            votes: 105,
+            article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+        }
+
+        return request(app)
+            .patch('/api/articles/1')
+            .send(voteUpdate)
+            .expect(201)
```

Comment:
Is the article created? That is what the 201 code means. Patch by it's very definition needs to have something already created to be able to update so a 200 status is more appropriate 👍